packages/sqlite/src/index.ts

```diff
@@ -118,115 +92,91 @@ class DatabaseConnectionImplementation implements DatabaseConnection {
 
   async tx<T>(fn: (db: DatabaseTransaction) => Promise<T>): Promise<T> {
     return this._mutex.writeLock(async () => {
-      await new Promise<void>((resolve, reject) => {
-        this._database.run('BEGIN', (err) => {
-          if (err) reject(err);
-          else resolve();
-        });
-      });
+      // TODO extract those as class variables
+      // however I do not know how to get
```

Comment:
I'm not sure what you mean by this TODO? I think these private properties look like a good approach here.

Reply:
Sorry this is a leftover, I already did that change.